Content.Client/Administration/UI/Notes/NoteEdit.xaml.cs

```diff
@@ -187,6 +213,16 @@ private void OnSeverityChanged(OptionButton.ItemSelectedEventArgs args)
         SeverityOption.SelectId(args.Id);
     }
 
+    private void OnLengthChanged(OptionButton.ItemSelectedEventArgs args)
+    {
+        ExpiryLengthDropdown.SelectId(args.Id);
+    }
+
+    private void OnTextChanged(HistoryLineEdit.LineEditEventArgs args)
+    {
+        ParseExpiryTime();
```

Comment:
Could be neat to have it change between "60 minutes => 1 hour", but at the same time might be a bit annoying.